code/modules/mob/living/inhand_holder.dm

```diff
@@ -44,6 +44,18 @@
 /obj/item/clothing/head/mob_holder/proc/update_visuals(mob/living/L)
 	appearance = L.appearance
 
+/obj/item/clothing/head/mob_holder/on_thrown(mob/living/carbon/user, atom/target)
+	if((item_flags & ABSTRACT) || HAS_TRAIT(src, TRAIT_NODROP))
+		return
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, span_notice("You set [src] down gently on the ground."))
+		release()
+		return
+
+	var/mob/living/throw_mob = held_mob
```

Comment:
Is there a reason you're creating a new variable here instead of just returning held_mob?